temporal-sdk/src/test/java/io/temporal/workflow/shared/TestWorkflowWithCronScheduleImpl.java

```diff
@@ -53,8 +57,11 @@ public String execute(String testName) {
     if (count == null) {
       count = new AtomicInteger();
       retryCount.put(testName, count);
+      lastCompletionResults.put(testName, new HashMap<Integer, String>());
     }
+
     int c = count.incrementAndGet();
+    lastCompletionResults.get(testName).put(c, lastCompletionResult);
```

Comment:
Please remove the initialization above and change this `get` onto `computeIfAbsent`. Let's use what Java gives us to the whole extend.